### ISSUE TITLE
hmmm... oddly familiar

### ISSUE THREAD

**Issue by Droid-D3V** [Sxmurai/tenacity-4.0#2]
this is literally taken from corrosionclient/src lol

**Comment by Sxmurai** [comment]
I was told it was tenacity. If it isn't oh well it's a block game. I was
told it was bought from the devs for like 1.7k or something, but I could be
wrong.

**Comment by mdo992** [comment]
> told it was bought from the devs for like 1.7k or something, but I could be

yes, https://github.com/CorrosionClient/src/tree/main/Tenacity: "This repository contains the source code for Tenacity client. A developer was paid $1,300 USD to share the source code from @2j9."

**Comment by Sxmurai** [comment]
Yeah, but I got it from somewhere else

**Comment by Sxmurai** [comment]
oh i just realized what you were tryna say, no i didnt steal it from the repo i got it in a zip file on discord and uploaded it, i had no clue of the other repos existence until you accused me of a reupload
